tests/AppBundle/Referent/ReferentDatabaseDumperTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Tests\AppBundle\Referent;
+
+use AppBundle\DataFixtures\ORM\LoadAdherentData;
+use AppBundle\DataFixtures\ORM\LoadEventData;
+use AppBundle\DataFixtures\ORM\LoadNewsletterSubscriptionData;
+use AppBundle\Referent\ReferentDatabaseDumper;
+use Tests\AppBundle\SqliteWebTestCase;
+
+class ReferentDatabaseDumperTest extends SqliteWebTestCase
+{
+    /**
+     * @var ReferentDatabaseDumper
+     */
+    private $dumper;
+
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp /^The "foobar" export type is unsupported/
+     */
+    public function testDumpWithUnsupportedFileType()
+    {
+        $this->dumper->dump(LoadAdherentData::ADHERENT_8_UUID, 'foobar');
+    }
+
+    /**
+     * @dataProvider provideFileType
+     */
+    public function testDumpFileType(string $type)
+    {
+        $path = $this->getDatabaseDumpedFilePath($type);
+
+        $this->assertFileNotExists($path);
+
+        $this->dumper->dump(LoadAdherentData::ADHERENT_8_UUID, $type);
+
+        $this->assertFileExists($path);
+        $this->assertGreaterThan(0, filesize($path));
+    }
```

Comment:
Could you also test it's valid JSON/serialized data? It could avoid having errors in production.